tests/plugins/mocha.test.support/hooks/hook.js

```diff
@@ -78,3 +79,28 @@ async function wakeDevices(logger, builder) {
 		await wake(builder.deviceId);
 	}
 }
+
+async function enableAutoRotation(logger, builder) {
+
+	async function autoRotate(device) {
+		const deviceId = device !== 'emulator' ? [ '-s', device ] : [];
+
+		// Enable auto-rotation if currently disabled.
+		const rotationStatus = await adb([ ...deviceId, 'shell', 'settings', 'get', 'system', 'accelerometer_rotation' ]);
+		if (rotationStatus && rotationStatus === '0') {
+			logger.info(`Enabling auto-rotation for ${device}`);
+
+			await adb([ ...deviceId, 'shell', 'settings', 'set', 'system', 'accelerometer_rotation', '1' ]);
```

Comment:
```suggestion
			await adb([ ...deviceId, 'shell', 'settings', 'put', 'system', 'accelerometer_rotation', '1' ]);
```

I tried this on the actual box and did find the setting was `'0'` and then tried this command and it said:
`Invalid command: set`
After some googling, I realized it should be `put`, which worked manually.